addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15185,13 +15194,13 @@ msgstr ""
 #. Description of setting "Pictures -> Resolution" with label #169
 #: system/settings/settings.xml
 msgctxt "#36310"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #. Description of settings category "Pictures -> Slideshow" with label #108
 #: system/settings/settings.xml
 msgctxt "#36311"
-msgid "Category containing settings for how picture slideshows are handled."
+msgid "Category containing settings for how the picture slideshow is handled."
```

Comment:
I went even further, and checked all entries that mention slideshow in any given sentence, no a single one was incorrect and all only refer to a single slideshow